### PR TITLE
[2.1.x] Added internal check before calling a validator

### PR DESCRIPTION
**Pull request by KorsaR-ZN**

### CODE REVIEW

File: phalcon/validation.zep [hunk 1]
```diff
@@ -459,4 +466,20 @@ class Validation extends Injectable implements ValidationInterface
 
 		return value;
 	}
+
+	/**
+	 * Internal validations, if it returns true, then skip the current validator
+	 */
+	protected function preChecking(string field, <ValidatorInterface> validator) -> boolean
+	{
+		if validator->getOption("allowEmpty", false) === true {
```
Comment (sergeyklay) [inline phalcon/validation.zep]: This means that the developer will not be able to use
```php
'allowEmpty' => 1
```

Comment (KorsaR-ZN) [inline phalcon/validation.zep]: @sergeyklay fixed it

**Comment by andresgutierrez** [comment]
Could you please rebase?

**Comment by KorsaR-ZN** [comment]
@andresgutierrez rebased

**Comment by andresgutierrez** [comment]
Thanks

**Pull request merged**